### ISSUE TITLE
Complete freeze of IntelliJ during connect

### ISSUE THREAD

**Issue by Hades32** [dboissier/mongo4idea#239]
Absolutely nothing works in IntelliJ while I wait e.g. for a connection timeout

**Comment by antonkomarev** [comment]
Yes, I can confirm that in case when `mongod` is started and you'll try to connect to server - all IDE instances will freeze for a 30 seconds.
